assets/scripts/aircraft.js

```diff
@@ -334,10 +334,16 @@ var Model=Fiber.extend(function() {
 zlsa.atc.AircraftFlightManagementSystem = Fiber.extend(function() {
   return {
     init: function(options) {
-      this.aircraft = options.aircraft;
```

Comment:
Fixed in new branch (complex-SIDs2).

I left it as removed, and created a new property for each aircraft, an "entity id", or "eid". This is set at spawn (and shifted as planes are removed) to be the index of a given aircraft within `prop.aircraft.list`. This makes it VERY convenient and easy to access different aircraft regardless of scope. When the fms is initialized, it also keeps a property `my_aircrafts_eid`, which can be used to refer back to the fms's parent object: the aircraft itself.